config/settings.yml

```diff
@@ -1,5 +1,5 @@
 active_record:
-  batch_size: 5000
+  batch_size: 50000
```

Comment:
I think we'll need to take a look at performance, but I have no reason to believe that a batch size of 50k is preferable.  That would lead to roughly 65mb insert statements

Reply:
These graphs might help us arrive at a number http://grafana.vfs.va.gov/d/rk7GzYxWk/ww-gids-outage-postmortem?orgId=1&var-datasource=Prometheus%20(Utility)&from=1574350105214&to=1574357153057